### PR TITLE
Update Podfile & Podspec.

### PR DESCRIPTION
**Pull request by SPaluchiewiczSofomo**
Use strict versions for dependencies.
Update LICENSE record.

I think we should also set:
- author - at this moment it's set to "author" => "author@domain.cn" 
(if we don't want to set email we can set only `Magic Leap`)
- version - at this moment it's set to 0.1.0 
(should this reflect npm package version)

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
-platform :ios, '11.0'
+platform :ios, '12.0'
 
 target 'RNMagicScriptHostApplication' do
   use_frameworks!
 
-  pod 'GLTFSceneKit'
+  pod 'GLTFSceneKit', '0.1.3'
   pod 'GrowingTextView', '0.7.1'
```
Comment (pleszkiewicz) [inline ios/Podfile]: Should be `0.7.2`?

File: ios/RNMagicScript.podspec [hunk 2]
```diff
@@ -5,8 +5,7 @@ Pod::Spec.new do |s|
   s.summary      = "RNMagicScript"
   s.description  = "Magic script components for React Native."
   s.homepage     = "https://www.magicscript.org/"
-  s.license      = "MIT"
-  # s.license    = { :type => "MIT", :file => "FILE_LICENSE" }
+  s.license    = { :type => "Apache License, Version 2.0", :file => "../LICENSE" }
   s.author       = { "author" => "author@domain.cn" }
```
Comment (pleszkiewicz) [inline ios/RNMagicScript.podspec]: "Magic Leap"

File: ios/RNMagicScript.podspec [hunk 3]
```diff
@@ -5,8 +5,7 @@ Pod::Spec.new do |s|
...
   s.platform     = :ios, "12.0"
   s.source       = { :git => "git@github.com:magic-script/magic-script-components-ios.git", :tag => "master" }
```
Comment (pleszkiewicz) [inline ios/RNMagicScript.podspec]: ios -> react-native

File: ios/RNMagicScript.podspec [hunk 4]
```diff
@@ -5,8 +5,7 @@ Pod::Spec.new do |s|
   s.summary      = "RNMagicScript"
   s.description  = "Magic script components for React Native."
```
Comment (pleszkiewicz) [inline ios/RNMagicScript.podspec]: MagicScript Components renderer for React Native platform.

**Comment by SPaluchiewiczSofomo** [comment]
@kpiascik due to new version of ChromaColorPicker causes build issue I'm going to merge it without your review - I'll handle your change request in separate PR.

**Pull request merged**